src/GraphQL.Tests/DocumentWritersTestData.cs

```diff
@@ -10,7 +10,6 @@ public class DocumentWritersTestData : IEnumerable<object[]>
             new SystemTextJson.DocumentWriter(new System.Text.Json.JsonSerializerOptions
             {
                 WriteIndented = true,
-                PropertyNamingPolicy = System.Text.Json.JsonNamingPolicy.CamelCase,
```

Comment:
CamelCase was and is not the default.  Tests need to examine the default situation.

Reply:
What will be the default naming policy after this? I thought camelCase was the default because when I define the field like this:
```
Field<DateTimeGraphType>("ThisIsADateTime");
```

The response is:
```
{"data":{"test":{"thisIsAString":"String Value"}}}
```

Reply:
The schema has a default naming policy of camelCase. This occurs within the schema before it hits the serializer